docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: "3.9"
+    
+services:
+  waltz:
+    image: ghcr.io/ljubon/waltz:postgresql 
+    restart: always
+    command: ["update","run"]
+    ports:
+      - "8080:8080"
+    depends_on:
+      - postgres
+    environment:
+      DB_HOST: "postgres"
+      DB_PORT: "5432"
+      DB_NAME: "waltz"
+      DB_USER: "waltz"
+      DB_PASSWORD: "waltz"
+
+  postgres:
+    image: postgres:10.6
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U waltz -d waltz"]
+      interval: 15s
+      timeout: 10s
+      retries: 5
```

Comment:
I would remove those as they are not truly used by `docker-compose` anyway, apart from displaying whether a service is ready (I might be totally wrong about this!)